storage/indexer_schemas/src/schema/indexer_metadata/mod.rs

```diff
@@ -41,5 +44,28 @@ impl ValueCodec<IndexerMetadataSchema> for MetadataValue {
     }
 }
 
+define_pub_schema!(TailerMetadataSchema, Version, (), TAILER_METADATA_CF_NAME);
```

Comment:
hmm.. it seems unnecessary to store an empty entry on each version just to keep track of the version?

either figure that out by seeking the actual data CF, or overwrite a single entry of metadata / progress entry seems better.